orchestration/hca_orchestration/config/resources/live_slack_client/global.yaml

```diff
@@ -1,3 +1,3 @@
-channel: "#monster-ci"
+channel: "#arh_testing_ingest"
```

Comment:
Wasn't sure if you meant to leave this change in

Reply:
Nope, fixed. Good eye!